webpack.config.prod.js

```diff
@@ -41,14 +29,17 @@ module.exports = merge(commonConfig, {
         cache: true,
         sourceMap: true,
       }),
-      new CompressionWebpackPlugin({
-        filename: '[path][base].gz',
-        algorithm: 'gzip',
-        test: /\.js$|\.css$|\.html$|\.eot?.+$|\.ttf?.+$|\.woff?.+$|\.svg?.+$/,
-        threshold: 10240,
-        minRatio: 0.8,
-      }),
-
+      ...(!process.env.IPFS_DEPLOY
```

Comment:
gzip not working with ipfs?

Reply:
gzip can be served only by server, IPFS build is serverless, and gzip just waste the space of cluster